src/absil/ilwritepdb.fs

```diff
@@ -330,17 +330,22 @@ let generatePortablePdb (embedAllSource:bool) (embedSourceList:string list) (sou
                     | None -> Array.empty<PdbSequencePoint>
                     | Some (_,_) -> minfo.SequencePoints
 
-            let getDocumentHandle d =
-                if docs.Length = 0 || d < 0 || d > docs.Length then
-                    Unchecked.defaultof<DocumentHandle>
-                else 
-                    match documentIndex.TryGetValue(docs.[d].File) with
-                    | false, _ -> Unchecked.defaultof<DocumentHandle>
-                    | true, h -> h
-
-            if sps.Length = 0 then
+            let builder = new BlobBuilder()
+            builder.WriteCompressedInteger(minfo.LocalSignatureToken)
+
+            if sps = Array.empty then
```

Comment:
Minor nit - I personally feel it's better to test to check for ``sps.Length = 0`` as in the original code.  

Relying on array equality is ok, but 

(1) forces an ``equality`` constraint on the element type of the array, which may not always hold, and 
(2) may be subtle to people coming from .NET/C# where array equality is reference equality